### DISCUSSION TITLE
Can i Customize the Elsa database Or Use my Own Theme

### DISCUSSION THREAD

**Discussion by asimp51** [elsa-workflows/elsa-core#618] [Q&A]

**Comment by sfmskywalker** [comment]
1) That depends on the level of customization you would like. Elsa 2 abstracts away data access, which means there are a number of persistence providers you can use and configure. If there are certain aspects of a provider you wish you could configure but isn't currently made available, we can have a look and see if we can change that.

2) Assuming you are referring to the Elsa Dashboard MVC Area, that is not themeable. For Elsa 2, I intend to provide an Orchard Core module wrapping Elsa. Orchard Core is a modular application framework that has themeability out of the box. Elsa 2 will initially be released with a Dashboard application built with Blazor. It might evolve into a modular application framework with theming, but there are no concrete plans for that yet. Elsa 2 will also come with a re-hostable workflow designer in the form of an HTML web component / custom element. This means you could build your own dashboard application which re-hosts the designer. The designer won't deal with managing workflow definition and instance lists (i.e. it won't have CRUD operations), so that is something you would build yourself by querying the Elsa 2 REST APIs
